js/interface-lists.js

```diff
@@ -224,10 +224,14 @@ function attahObservers() {
           }
         }
 
-        return Promise.all([
-          linkAddEntryProvider.forwardSaveRequest(),
```

Comment:
`forwardSaveRequest` does not return a promise https://github.com/Fliplet/fliplet-api/blob/afc71fa4edfbac0e8ff568141a8a4dd6a34573dc/public/assets/fliplet-core/1.0/core.js#L1173-L1181